lighthouse-core/gather/gather-runner.js

```diff
@@ -348,6 +351,7 @@ class GatherRunner {
     // Nest LighthouseRunWarnings, if any, so they will be collected into artifact.
     const uniqueWarnings = Array.from(new Set(gathererResults.LighthouseRunWarnings));
     gathererResults.LighthouseRunWarnings = [uniqueWarnings];
+    gathererResults.fetchedAt = [gathererResults.fetchedAt];
```

Comment:
Works now, sorry for delaying. Extremely hungover yesterday 😄

Reply:
Another thing, if people use this through the CLI, do we need to expose name etc in order to save it to disk using `-A` or `-G`?